lib/server/controller.js

```diff
@@ -198,6 +198,7 @@ class RangerController {
     dm.once('error', ( err ) => {
       dm.removeAllListeners();
       this.errorResponse(this.response, 500, err);
+      throw new Error(err);
```

Comment:
We decided that if the cursor.next() callback in DataMapper was called with an error, not to handle the error and close the server. Throwing the error here will cause an unhandled exception to https://github.com/aacn500/npg_ranger/blob/f2730567456208561fe81bad72329292e5bc1548/lib/server.js#L225 and gracefully close the server with error logging and error code.